### DISCUSSION TITLE
visual bug in fullscreen mode

### DISCUSSION THREAD

**Discussion by chanleung489** [ppy/osu#19704] [Q&A]
### Type

Game behaviour

### Bug description

When osu is in fullscreen, it stay normal for a few seconds, then the bug happens.
it only happens when fullscreen

screen is covered by weird vertical stripes of red, green, and blue and color is distorted. 

screen is shifted to the right and loops back on the left (which means when cursor is moved to the right edge of the screen it actually shows up on the left). 

But it looks normal when captured by obs or discord screen sharing.

im on a samsung galaxy book pro with i7-1165G7 and iris xe grpahics
intel graphics command center say im using the most current driver

### Screenshots or videos

![圖片](https://user-images.githubusercontent.com/110078497/184062457-330772d0-f408-4662-b4c9-d1889094a904.png)

![20220811_113314](https://user-images.githubusercontent.com/110078497/184061841-381865a1-e67f-43ed-9b65-5d3de01c029f.jpg)

![IMG_20220811_114333](https://user-images.githubusercontent.com/110078497/184061947-68c8f9c6-a770-4fc3-8486-4d4f4232a80d.jpg)

![20220811_114511](https://user-images.githubusercontent.com/110078497/184061954-3598aec5-6da7-4aad-9ddf-38049fa65f85.jpg)

video
https://youtu.be/b7dn20lJdcQ

### Version

2022.731.1

### Logs

[database.log](https://github.com/ppy/osu/files/9305957/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9305958/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9305959/network.log)
[performance.log](https://github.com/ppy/osu/files/9305960/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9305961/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9305962/updater.log)

**Comment by peppy** [comment]
Please update your graphics drivers.
